README.md

```diff
@@ -17,39 +17,38 @@ Daffodil is a frontend toolkit that intends to accomplish three things:
 
 * :free: Forever Free, Open Source, and MIT Licensed
 * :hammer: Platform Agnostic Drivers For Your Platform of Choice
-    * Shopify
-    * Magento 2
-    * In-Memory Backend
-        * Prebuilt
+    * Shopify (WIP, 5% coverage)
+    * Magento 2 (WIP, 50% coverage)
+    * In-Memory Backend (60% coverage)
```

Comment:
What does "coverage" mean here?

Reply:
I intended coverage to mean the 5% complete of a "stock" ecommerce site. I agree that this is vague.

Reply:
Maybe "implemented" would be a better word.

Reply:
I've updated this.